app/src/main/java/com/github/globant/githubsubscribers/subscriberslist/interactor/SubscribersListInteractorImpl.java

```diff
@@ -24,8 +25,13 @@ public void getSubscribersDataList(final OnFinishedListener listener) {
         call.enqueue(new Callback<List<Subscriber>>() {
             @Override
             public void onResponse(Call<List<Subscriber>> call, Response<List<Subscriber>> response) {
-                List<Subscriber> userList = response.body();
-                listener.onResponse(userList);
+                if(response.isSuccessful()){
+                    List<Subscriber> userList = response.body();
+                    listener.onResponse(userList);
+                }
+                else{
+                    listener.onFailure(Constants.MESSAGE_FAILED_SERVICE);
```

Comment:
This part will be to get better by Edwin, he will implement a best way for manage the errors in the next PR, because he started this task